src/bitcoin-tx.cpp

```diff
@@ -104,10 +104,10 @@ static int AppInitRawTx(int argc, char* argv[])
             "\n";
         strUsage += gArgs.GetHelpMessage();
 
-        fprintf(stdout, "%s", strUsage.c_str());
+        tfm::format(std::cout, "%s", strUsage.c_str());
 
         if (argc < 2) {
-            fprintf(stderr, "Error: too few parameters\n");
+            tfm::format(std::cerr, "Error: too few parameters\n");
```

Comment:
nit: a few of these could drop formatting altogether, via `<<`

Reply:
I believe tfm uses `<<` under the hood, so it shouldn't matter.

Reply:
Yeah, the output should be the same, just would remove a layer of indirection.

Reply:
that's just a matter of style; it seems most of the developers of this code base prefer %X formatting to the various `<<` incantations (hence the use of `tinyformat` in the first place), so consistently using `tfm::format` seems good to me

Reply:
Fair enough. Just a nit.